tools/autoconf/Makefile

```diff
@@ -15,23 +15,15 @@ PKG_HASH:=64ebcec9f8ac5b2487125a86a7760d2591ac9e1d3dbd59489633f9de62a57684
 
 include $(INCLUDE_DIR)/host-build.mk
 
-HOST_CONFIGURE_ARGS += \
-	--datarootdir=$(STAGING_DIR_HOST)/share
-
 HOST_CONFIGURE_VARS += \
 	PERL="/usr/bin/env perl"
 
 define Host/Compile
-	export SHELL="$(BASH)"; $(MAKE) -C $(HOST_BUILD_DIR)
-endef
-
-define Host/Install
-	export SHELL="$(BASH)"; $(MAKE) -C $(HOST_BUILD_DIR) install
+	$(call Host/Compile/Default)
```

Comment:
`define Host/Compile` is now doing the same as the default one. I think this is not needed any more.

Reply:
I add a line there in the commit after, "tools/autoconf: bump to 2.71"

one of the things that changed was how the manual is generated.
that version fails to build on the CI because it doesn't have help2man, so we skip generating the man pages instead.

it's a nicer way to achieve the same result, which I tested sometime last year, but it was ugly and complicated

https://github.com/mpratt14/openwrt/commit/5caa888eebde032500faa54b4e52bd0144416bf8

Reply:
Sorry I looked at the cleanup commit 24bf39833b6b745c0f385ca3a8a9c6b686c85e6b and did not saw that you add the extra line some patches later.